.github/workflows/ci.yml

```
@@ -14,23 +14,30 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v4
 
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)